server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/deltas.ts

```diff
@@ -147,6 +148,20 @@ export function create(
 			const from = stringToSequenceNumber(request.query.from);
 			const to = stringToSequenceNumber(request.query.to);
 			const tenantId = getParam(request.params, "tenantId") || appTenants[0].id;
+			const caller = request.query.caller?.toString();
+
+			const clientRequestSize = to - from - 1;
+			if (clientRequestSize > getDeltasRequestMaxOpsRange) {
+				return handleResponse(
+					Promise.reject(
+						new NetworkError(
+							400,
+							`Requested too many ops. Max ops range is: ${getDeltasRequestMaxOpsRange}`,
+						),
+					),
+					response,
+				);
+			}
```

Comment:
Nit, could we create a middleware like `validateRequestParams` to validate the sn range? Of course, if it leads duplicated code like parsing the `from` `to`, then no need to be that picky.